example-deployment/docker/builder.dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ros:foxy-ros-base-focal
+
+RUN apt-get update && apt-get install -y curl && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && apt-get install -y nodejs python3-pip
+
+RUN pip3 install pipenv
+
+COPY rmf/rmf_internal_msgs/rmf_charger_msgs /root/rmf_ws/src/rmf_dispenser_msgs
+COPY rmf/rmf_internal_msgs/rmf_dispenser_msgs /root/rmf_ws/src/rmf_dispenser_msgs
+COPY rmf/rmf_internal_msgs/rmf_door_msgs /root/rmf_ws/src/rmf_door_msgs
+COPY rmf/rmf_internal_msgs/rmf_fleet_msgs /root/rmf_ws/src/rmf_fleet_msgs
+COPY rmf/rmf_internal_msgs/rmf_ingestor_msgs /root/rmf_ws/src/rmf_ingestor_msgs
+COPY rmf/rmf_internal_msgs/rmf_lift_msgs /root/rmf_ws/src/rmf_lift_msgs
+COPY rmf/rmf_internal_msgs/rmf_task_msgs /root/rmf_ws/src/rmf_task_msgs
+COPY rmf/rmf_internal_msgs/rmf_traffic_msgs /root/rmf_ws/src/rmf_traffic_msgs
+COPY rmf/rmf_internal_msgs/rmf_workcell_msgs /root/rmf_ws/src/rmf_workcell_msgs
+COPY rmf/traffic_editor/building_map_msgs /root/rmf_ws/src/building_map_msgs
```

Comment:
This might have to change to something like `rmf/rmf_building_map_msgs /root/rmf_ws/src/rmf_building_map_msgs` due to a recent [refactoring](https://github.com/open-rmf/rmf_traffic_editor/pull/308)